kernel/src/actions/mod.rs

```diff
@@ -442,6 +442,12 @@ struct Remove {
     #[cfg_attr(test, serde(skip_serializing_if = "Option::is_none"))]
     pub(crate) size: Option<i64>,
 
+    /// Contains [statistics] (e.g., count, min/max values for columns) about the data in this logical file.
```

Comment:
nit: maybe some more useful context (feel free to add above too)
```suggestion
    /// Contains [statistics] (e.g., count, min/max values for columns) about the data in this logical file encoded as a JSON string.
```